src/main/scala/firrtl/passes/Inline.scala

```diff
@@ -353,8 +358,11 @@ class InlineInstances extends Transform with DependencyAPIMigration with Registe
         Some(m.map(onStmt(ModuleName(m.name, CircuitName(c.main)))))
     })
 
-    val renames = renamesSeq.tail.foldLeft(renamesSeq.head)(_ andThen _)
+    val renames = renamesSeq match {
+      case Nil        => None
+      case car :: cdr => Some(cdr.foldLeft(car)(_ andThen _))
```

Comment:
```suggestion
    val renames = if (renamesSeq.isEmpty) None else renamesSeq.tail.foldLeft(renamesSeq.head)(_ andThen _)
```
Actually this is probably better

Reply:
You would need to make the other case an option (it's not in your suggestion).

If we're golfing...

`renamesSeq.reduceLeftOption(_ andThen _)`

Reply:
Right yeah, thanks